ci-operator/step-registry/openshift-extended/test/openshift-extended-test-commands.sh

```diff
@@ -540,4 +564,5 @@ function check_case_selected {
         echo "do not find case"
     fi
 }
+remove_admin_user
 run
```

Comment:
It needs to be run before e2e cases are run, i.e. before the call line of `extended-platform-tests`.

Reply:
Given you would repush, I suggest rename the function to be a read-i.e.-clear `remove_kubeadmin_user`.

Reply:
Updated, Thanks